### PR TITLE
Verification test for arm64

### PR DESCRIPTION
**Pull request by deepsm007**
/cc @yselkowitz

### CODE REVIEW

**Comment by yselkowitz** [comment]
This is still a draft, is something still missing here?

**Comment by deepsm007** [comment]
> This is still a draft, is something still missing here?

Waiting for confirmation from @yselkowitz and @andymcc if we want this verification test running in CI

**Comment by yselkowitz** [comment]
If the testsuite is working, relevant to ARM, and in use on x86, then I think we'd want it.  cc @aleskandro as this pertains to cucushift on ARM

**Comment by aleskandro** [comment]
Hello, AFAIK, this is related to [OCPQE-5515](https://issues.redhat.com/browse/OCPQE-5515) and part of the OKRs of the openshift-qe org. It went out of the ARM QE scope for 4.10 due to the efforts on multi-arching verification-tests and cucushift for 4.10 ARM GA. The test suite is currently running in the QE CI Jenkins infra as part of the QE validations.

We'd like to come back at it so to finally integrate ARM QE jobs in Prow and unify devs/QE CIs. Moreover, we should consider the work done in [ci-tools#2724](https://github.com/openshift/ci-tools/pull/2724#discussion_r829807440). Asking @yunjiang29 for confirmation about that. Per my understanding, these jobs should use the QE cluster profiles from now on. 

Leaving some comments in the code before merging. 

Thanks @deepsm007 for this PR.

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10.yaml [hunk 1]
```diff
@@ -36,6 +42,21 @@ tests:
     - chain: ipi-aws-pre
     - ref: cucushift-pre
     workflow: cucushift-e2e
+- as: e2e-aws-arm64-ipi
+  interval: 24h
+  steps:
+    cluster_profile: aws-arm64
```
Comment (aleskandro) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10.yaml:48]: Should this be the QE AWS profile @yunjiang29 ?

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10.yaml [hunk 2]
```diff
@@ -36,6 +42,21 @@ tests:
...
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: arm-ci.devcluster.openshift.com
```
Comment (aleskandro) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10.yaml:52]: We'd add the following two env variables to query the test cases in cucushift.
```
      E2E_RUN_TAGS: '@aws-ipi'
      TAG_VERSION: '@4.10'
```
Others could be needed. I'll give confirmation ASAP

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10.yaml (hunk 1 above)
Comment (yunjiang29) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10.yaml:48]: if it's part of QE profiles, the answer is yes.

@aleskandro is there any special credentials required? or just pull secret and AWS credential is enough?

Comment (aleskandro) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10.yaml:48]: @yunjiang29 no special credentials. We use the same AWS accounts for both x86 and arm testing in Jenkins.

Comment (yunjiang29) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10.yaml:48]: OK , `aws-qe` is ready now, use it if required.

Comment (deepsm007) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10.yaml:48]: The `aws-qe` won't work with arm64. The quota-slice for each architecture is different. For reference -> https://github.com/openshift/release/blob/master/core-services/prow/02_config/_boskos.yaml#L174-L210 
Available options:
1. Add a quota-slice for `aws-arm64-qe`
2. Use the same quota-slice as of arm64 CI `aws-arm64`

**Comment by deepsm007** [comment]
Followed-up by QE

**Pull request closed without merging**